components/footer/footer.module.scss

```diff
@@ -0,0 +1,5 @@
+.infoRepo {
+  font-weight: 100;
+  margin: 0 auto;
+  text-align: center;
```

Comment:
Just add top and bottom padding of 15px.

Reply:
Done, Could you please check